samples/boards/nrf52/power_mgr/README.rst

```diff
@@ -0,0 +1,58 @@
+.. _power-mgr-sample:
+
+Power management demo
+#####################
+
+Overview
+********
+
+A sample implementation of a power manager app that uses the Zephyr
+power management infrastructure.
+
+This app will cycle through the various power schemes at every call
+to _sys_soc_suspend() hook function.
+It will cycle through the following states:
+
+1. CPU Low Power State
+   A. CONSTANT LATENCY LOW POWER MODE : This Low Power State triggers CONSTLAT
+   task on nrf SOC. In this mode Low Exit Latency. In this Mode 16MHz Clock is
+   kept on.
+   B. LOW POWER STATE : In this Power State CONSTLAT task is triggered on nrf
+   SOC. This is deeper power state than CONSTANT LATENCY Mode, In this mode
+   16MHz Clock is turned off and only 32KHz clock is retained.
+
+2. Deep Sleep - This Power State is mapped to System OFF state. In this Mode
```

Comment:
In this mode all devices on board are suspended and the SOC
     and all devices lose context on wake up.

Capital letters should be lowercase.